deploy/deployment.yaml

```diff
@@ -29,6 +29,11 @@ spec:
           value: <application_base_url>
         - name: INSTALL_OPTIONAL_LIBS
           value: "mongodb,mysql,influxdb,snowflake,oracle,clickhouse,google"
+        - name: APPLICATIONINSIGHTS_CONNECTION_STRING
```

Comment:
What is the purpose of this variable?

Reply:
Integrating Azure Application Insights with Bold BI Enterprise Edition enables you to track and visualize your application’s performance, detect issues, and enhance your application’s overall reliability.

Reply:
Hi @am-tribia ,
This environment variable is used for integrating Azure Application Insights with Bold BI Enterprise Edition. It enables you to track and visualize your application’s performance, detect issues, and enhance your application’s overall reliability.